pages/dkp/kaptain/2.0.0/install/prerequisites/index.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: layout.pug
+navigationTitle: Prerequisites
+title: Prerequisites for installing Kaptain
+menuWeight: 2
+excerpt: Ensure these prerequisites are met before installing Kaptain
+beta: false
+enterprise: false
+---
+
+- [`kubectl`][kubectl] on your installation machine 
+
+- A DKP cluster, either one of the following, depending on your license 
+
+  - An **Essential cluster** for customers with an [Essential license](../../../../kommander/2.2/licensing/essential/)
+  - A **Managed** or **Attached** cluster for customers with an [Enterprise license](../../../../kommander/2.2/licensing/enterprise/), in which case, said cluster needs to be the only one in its workspace
+
+- The following **Platform applications** (dependencies) enabled in the cluster where you want to install Kaptain:
+
+  - **Istio**
+  - **MinIO Operator** (installed by default in some clusters, not on attached clusters) 
+  - **Knative**, if you are running the [serverless installation of KServe](https://kserve.github.io/website/0.9/admin/serverless/), which is the default mode that comes pre-bundled with Kaptain. (Not required if you are using the [RawDeployment installation mode](https://kserve.github.io/website/0.9/admin/kubernetes_deployment/))
```

Comment:
I wonder if we'll constantly have to update this kserve link to point to the current latest version.
same comment applies to the corresponding wiki page on confluence

Reply:
Probably we will. I added a comment in confluence and a label, so we can hopefully add this to the list of things to be checked for each release.